src/components/ModularTable/ModularTable.tsx

```diff
@@ -107,6 +108,7 @@ function ModularTable({
 }
 
 ModularTable.propTypes = {
+  className: PropTypes.string,
```

Comment:
I don't think this change is quite right, passing `className` to a ModularTable doesn't seem to apply the class.

Reply:
Ah oops, brainfart. Added a hardcoded className and thought "yeah that's totally a prop".

I think I'll leave out being able to pass a `className` prop and just have the hardcoded wrapper className.